api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -356,70 +355,100 @@ public void getDateFormat_shouldReturnAPatternWithFourYCharactersInIt() {
-		Assert.assertFalse(OpenmrsUtil.containsUpperAndLowerCase(null));
+		String str = "HELLO";
+		Assert.assertFalse("Hello",
+				OpenmrsUtil.containsUpperAndLowerCase("HELLO"));
+		Assert.assertFalse("THE NUMBER 10?",
+				OpenmrsUtil.containsUpperAndLowerCase("THE NUMBER 10?"));
+		Assert.assertFalse(".", OpenmrsUtil.containsUpperAndLowerCase(""));
+		Assert.assertFalse(null, OpenmrsUtil.containsUpperAndLowerCase(null));
 	}
 	
 	/**
 	 * @see OpenmrsUtil#containsUpperAndLowerCase(String)
 	 */
 	@Test
 	public void containsUpperAndLowerCase_shouldReturnFalseIfStringDoesNotContainUpperCaseCharacters() {
-		Assert.assertFalse(OpenmrsUtil.containsUpperAndLowerCase("hello"));
-		Assert.assertFalse(OpenmrsUtil.containsUpperAndLowerCase("the number 10?"));
-		Assert.assertFalse(OpenmrsUtil.containsUpperAndLowerCase(""));
-		Assert.assertFalse(OpenmrsUtil.containsUpperAndLowerCase(null));
+		String str = "hello";
+
+		Assert.assertFalse("hello",
+				OpenmrsUtil.containsUpperAndLowerCase("hello"));
+		Assert.assertFalse("the number 10?",
+				OpenmrsUtil.containsUpperAndLowerCase("the number 10?"));
+		Assert.assertFalse("", OpenmrsUtil.containsUpperAndLowerCase(""));
+		Assert.assertFalse(null, OpenmrsUtil.containsUpperAndLowerCase(null));
 	}
 	
 	/**
 	 * @see OpenmrsUtil#containsOnlyDigits(String)
 	 */
 	@Test
 	public void containsOnlyDigits_shouldReturnTrueIfStringContainsOnlyDigits() {
-		Assert.assertTrue(OpenmrsUtil.containsOnlyDigits("1234567890"));
+		String str = "1234567890";
+		Assert.assertTrue("1234567890",
+				OpenmrsUtil.containsOnlyDigits("1234567890"));
 	}
 	
 	/**
 	 * @see OpenmrsUtil#containsOnlyDigits(String)
 	 */
 	@Test
 	public void containsOnlyDigits_shouldReturnFalseIfStringContainsAnyNonDigits() {
-		Assert.assertFalse(OpenmrsUtil.containsOnlyDigits("1.23"));
-		Assert.assertFalse(OpenmrsUtil.containsOnlyDigits("123A"));
-		Assert.assertFalse(OpenmrsUtil.containsOnlyDigits("12 3"));
-		Assert.assertFalse(OpenmrsUtil.containsOnlyDigits(""));
-		Assert.assertFalse(OpenmrsUtil.containsOnlyDigits(null));
+		String str = "1.23";
+		Assert.assertFalse("1.23", OpenmrsUtil.containsOnlyDigits("1.23"));
+		Assert.assertFalse("1.23", OpenmrsUtil.containsOnlyDigits("123A"));
+		Assert.assertFalse("1.23", OpenmrsUtil.containsOnlyDigits("12 3"));
+		Assert.assertFalse("", OpenmrsUtil.containsOnlyDigits(""));
+		Assert.assertFalse(null, OpenmrsUtil.containsOnlyDigits(null));
 	}
 	
 	/**
 	 * @see OpenmrsUtil#containsDigit(String)
 	 */
 	@Test
 	public void containsDigit_shouldReturnTrueIfStringContainsAnyDigits() {
-		Assert.assertTrue(OpenmrsUtil.containsDigit("There is 1 digit here."));
+		String str = "There is 1 digit here.";
+		AssertTrue(OpenmrsUtil.containsDigit("There is 1 digit here."),
+				"There is 1 digit");
 	}
 	
 	/**
-	 * @see OpenmrsUtil#containsDigit(String)
+	 * @param containsDigit
+	 * @param string
+	 * @return
+	 */
+	private Object AssertTrue(boolean containsDigit, String string) {
+		return null;
+
+	}
+
```

Comment:
these blank spaces to need to be removed to

Reply:
ok am going to work on them